### PR TITLE
display.error on no matching --start-at-task

### PR DESCRIPTION
**Pull request by mkrizek**
##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
--start-at-task

##### ANSIBLE VERSION

```
devel
```

##### ADDITIONAL INFORMATION
https://github.com/ansible/ansible/issues/11692#issuecomment-400576327

### CODE REVIEW

**Comment by bcoca** [comment]
was there a specific error/example that prompted this fix?

**Comment by mkrizek** [comment]
@bcoca https://github.com/ansible/ansible/issues/11692#issuecomment-400576327

File: lib/ansible/executor/play_iterator.py [hunk 1]
```diff
@@ -232,6 +232,12 @@ def __init__(self, inventory, play, play_context, variable_manager, all_vars, st
             # play context to flag that we've started at a task (and future
             # plays won't try to advance)
             play_context.start_at_task = None
+        elif play_context.start_at_task:
```
Comment (bcoca) [inline lib/ansible/executor/play_iterator.py]: what if the task is on a subsequent play? Won't this error if it is not always in the first play?

Comment (mkrizek) [reply lib/ansible/executor/play_iterator.py]: Ah, yes. Fixed. Thanks!

**Pull request merged**